### PR TITLE
done tag

### PR DESCRIPTION
**Pull request by tuansaker1412**

### CODE REVIEW

**Comment by tuansaker1412** [comment]
ready

File: app/models/post.rb [hunk 1]
```diff
@@ -10,5 +10,16 @@ class Post < ApplicationRecord
   validates :content, presence: true
   validates :user_id, presence: true
 
+
   mount_uploader :picture, PictureUploader
+
+  def all_tags=names
```
Comment (huongnguyenmta) [inline app/models/post.rb]: `all_tags = names`

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -7,6 +7,8 @@
           <div class="col s12">
             <%= image_tag @post.picture, class: "responsive-img" if @post.picture? %>
           </div>
+          <h4 class="title"><%= @post.title %></h4>
+          <p>Tags: <%= @post.all_tags %></p>
```
Comment (huongnguyenmta) [inline app/views/posts/show.html.erb]: I18n

File: app/views/shared/_post_form.html.erb [hunk 3]
```diff
@@ -12,6 +12,11 @@
         <%= f.text_area :content, class: "materialize-textarea validate", id: "content" %>
         <%= f.label t ".compose", for: "content" %>
       </div>
+      <div class="input-field">
+        <i class="material-icons prefix">loyalty</i>
```
Comment (huongnguyenmta) [inline app/views/shared/_post_form.html.erb]: I18n

File: app/views/shared/_post_form.html.erb [hunk 4]
```diff
@@ -12,6 +12,11 @@
...
+        <%= f.text_field :all_tags, class: "validate chip", id: "tag" %>
+        <%= f.label "Tags", for: "tag" %>
```
Comment (huongnguyenmta) [inline app/views/shared/_post_form.html.erb]: I18n

**Comment by huongnguyenmta** [comment]
commented

**Comment by tuansaker1412** [comment]
ready

**Pull request merged**